cli/cli.go

```diff
@@ -213,6 +219,11 @@ func (il *AddressInputList) Set(value string) error {
 	return nil
 }
 
+// Type return a string representation of AddressInputList.
+func (il *AddressInputList) Type() string {
+	return "AddressInputList"
```

Comment:
Users won't know what `AddressInput` means. Let's replace with the syntax for it.
Please remove the syntax example from the help message

```suggestion
	return "([<host>]:<port>)..."
```